src/modules/storage/LedgerStorage.ts

```diff
@@ -2381,6 +2381,52 @@ export class LedgerStorage extends Storages {
         return this.query(sql, [address, address]);
     }
 
+    /**
+     * Provides a balance of address
+     * @param address The address to check the balance
+     */
+    public getWalletBalance(address: string): Promise<any[]> {
+        let sql = `
+            SELECT
+                ? as address,
```

Comment:
```suggestion
                T.address as address,
```

Reply:
When the data did not exist, the address came out null and was treated like that's